GooglePay/src/main/java/com/braintreepayments/api/googlepay/GooglePayClient.kt

```diff
@@ -263,14 +246,14 @@ class GooglePayClient @VisibleForTesting internal constructor(
      * @param callback    [GooglePayTokenizeCallback]
      */
     @SuppressWarnings("TooGenericExceptionCaught")
-    fun tokenize(paymentData: PaymentData?, callback: GooglePayTokenizeCallback) {
+    fun tokenize(paymentData: PaymentData, callback: GooglePayTokenizeCallback) {
```

Comment:
I think making `PaymentData` non-null would be a breaking change for merchants. Should we keep it as nullable and return an error for this case? Or should we keep this change and update the changelog?

Reply:
It looks like it's also an overload function to fun tokenize(paymentAuthResult: GooglePayPaymentAuthResult, callback: GooglePayTokenizeCallback), which checks non-null before calling this function tokenize(paymentData: PaymentData, callback: GooglePayTokenizeCallback). Inside this function, each try-catch block tries to access data paymentData until the final JSONException catch. When the passed parameter paymentData is null, it will always cause exception. We can only handle exception when the data in paymentData is invalid by making it non-null.

Reply:
Ahh I see. I missed that!